lib/custom/src/MShop/Service/Provider/Payment/Payone.php

```diff
@@ -22,6 +22,138 @@ class Payone
 	implements \Aimeos\MShop\Service\Provider\Payment\Iface
 {
 	/**
+	 * Returns the data passed to the Omnipay library
+	 *
+	 * @param \Aimeos\MShop\Order\Item\Base\Iface $base Basket object
+	 * @param $orderid Unique order ID
+	 * @param array $params Request parameter if available
+	 */
+	protected function getData( \Aimeos\MShop\Order\Item\Base\Iface $base, $orderid, array $params )
+	{
+		$urls = $this->getPaymentUrls();
```

Comment:
The better way than duplicating the code would be to call
```$data = parent::getData( $base, $orderid, $params );```
and add your own data to the returned array.